packages/core/src/components/tabs/inline-tabs/inline-tabs.scss

```diff
@@ -35,12 +34,12 @@
 
   .scroll-right-button {
     right: 0;
-    z-index: calc(tds-z-index(tab) + 1);
+    z-index: 1;
```

Comment:
Asking because I don't know - will the set z-index not lead to the scroll buttons ending up behind something else if the user has something in the surrounding context that has a higher z-index?